lib/install-dependencies.js

```diff
@@ -17,20 +18,23 @@ export default async function installDependencies(names) {
 		manifest = JSON.parse(packageJson);
 	} catch {
 		throw new Error(
-			"couldn't parse origami-ci package.json as JSON, ensure it's there and valid json!"
+			"couldn't parse origami-ci package.json as JSON, ensure it's there and is valid json!"
 		);
 	}
 
 	let specs = names.map(name => {
-		let version = manifest.peerDependencies[name];
+		let version = manifest._origamiGlobalDependencies[name];
 		if (!version) {
```

Comment:
Perhaps here we could also check to see if it does exist in the `dependencies` section and if it does, we can say something like.

"It looks like it has been added to the `dependencies` section, we avoid adding to this section for command dependencies to keep the installation time of `origami-ci` as small as possible. You will need to move the dependency to a section called `_origamiGlobalDependencies`".

Reply:
added! thanks!